modules/controlfreec/1.1/config/default.yaml

```diff
@@ -17,41 +17,53 @@ lcr-modules:
         options:
             configFile: "{MODSDIR}/config/freec/config_WGS.txt"
             BedGraphOutput: FALSE
-            breakPointThreshold: 0.8 #use 0.6 for more segments to get more CNVs
+            breakPointThreshold: 0.6 #use 0.6 for more segments to get more CNVs
             breakPointType: 2
-            coefficientOfVariation: 0.05
+            coefficientOfVariation: 0.062
             contamination: 0 # a priori known value of tumor sample contamination by normals
-            contaminationAdjustment: FALSE
-            degree: "3&4" #3&4 for WGS; 1 for WES
-            forceGCcontentNormalization: 0 #0 for WGS; 1 for WES
-            minCNAlength: 1 #1 for WGS; 3 for WES
+            contaminationAdjustment: TRUE
+            degree: '3\&4' #3&4 for WGS; 1 for WES
+            forceGCcontentNormalization: 1 #0 for WGS; 1 for WES # 0 forces control-base normalization, 1 forces GC
+            intercept: 0 # 0 for control-based (paired) ; 1 for GC-content (unpaired)
+            minCNAlength: 8 #1 for WGS; 3 for WES
+            minimalCoveragePerPosition: 10 # default is 0
+            minMappabilityPerWindow: 0.9
+            minQualityPerPosition: 20
             minimalSubclonePresence: 20 #20 for WGS; 30 for WES
             maxThreads: 24
-            noisyData: FALSE #set TRUE for exomes to avoid false positives due to non-uniform capture
-            ploidy: 2,3,4 #will select the ploidy that explains the most CNAs
+            noisyData: TRUE #set TRUE for exomes to avoid false positives due to non-uniform capture
+            ploidy: 2 #will select the ploidy that explains the most CNAs
+            shiftInQuality: 0
             #sex: XX or XY
-            step: 10000 # only if window is specified; do not use for WES
+            step: 20000 # only if window is specified; do not use for WES
```

Comment:
In case of WES, what this parameter should be? is it 0?

Reply:
Yes, for WES it is 0 and window will also be 0 since it uses the exons as dynamic windows